### PR TITLE
Carpolly

### PR DESCRIPTION
**Pull request by PollyV1**
## Description

Revised some information.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* Removed Via keymap

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/carpolly/keymaps/carpolly.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
+{
```
Comment (drashna) [inline keyboards/carpolly/keymaps/carpolly.json:1]: This appears to be a VIA json. 
https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs

> Do not include VIA json files in the PR. These do not belong in the QMK repository as they are not used by QMK firmware – they belong in the [VIA Keyboard Repo](https://github.com/the-via/keyboards>

File: keyboards/carpolly/keymaps/carpollyREV1.json [hunk 2]
```diff
@@ -0,0 +1,223 @@
+{
```
Comment (drashna) [inline keyboards/carpolly/keymaps/carpollyREV1.json:1]: this should be keymap.json, or removed.

File: keyboards/carpolly/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        LT3_TAB, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_COLN, KC_ENT,  
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_DOT,  KC_SLSH, KC_UP,   MO(1),   
+        KC_LCTL, KC_LGUI,  LT2_LAL, KC_SPC,  KC_LEFT, KC_DOWN, KC_RGHT  
+    ),
+
+    [1] = LAYOUT(
+        KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  
+        KC_CAPS, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_ENT,  
+        KC_LSFT, KC_MPLY, KC_VOLD, KC_VOLU, _______, _______, _______, KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, _______, 
+        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  KC_HOME, KC_PGDN, KC_PGUP
```
Comment (drashna) [inline keyboards/carpolly/keymaps/default/keymap.c:36]:
```suggestion
        KC_LCTL, KC_LGUI, KC_LALT,                   KC_SPC,                             KC_HOME, KC_PGDN, KC_PGUP  
```

File: keyboards/carpolly/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        KC_CAPS, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_ENT,  
+        KC_LSFT, KC_MPLY, KC_VOLD, KC_VOLU, _______, _______, _______, KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, _______, 
+        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  KC_HOME, KC_PGDN, KC_PGUP  
+    ),
+
+    [2] = LAYOUT(
+        RESET,   _______, _______, _______, _______, _______, _______, _______, _______, KC_MINS, KC_EQL,  KC_INS,  
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_LBRC, KC_RBRC, KC_BSLS, 
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_RSFT, _______, 
+        _______, _______, _______, _______, _______, KC_DEL,  KC_INS
```
Comment (drashna) [inline keyboards/carpolly/keymaps/default/keymap.c:43]:
```suggestion
        _______, _______, _______,                   _______,                            _______, KC_DEL,  KC_INS   
```

File: keyboards/carpolly/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_LBRC, KC_RBRC, KC_BSLS, 
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_RSFT, _______, 
+        _______, _______, _______, _______, _______, KC_DEL,  KC_INS   
+    ),
+
+    [3] = LAYOUT(
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, 
+        _______, _______, _______, KC_SPC,  _______, _______, _______
```
Comment (drashna) [inline keyboards/carpolly/keymaps/default/keymap.c:50]:
```suggestion
        _______, _______, _______,                   KC_SPC,                             _______, _______, _______  
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by drashna** [comment]
Also, duplicate of #15774

**Comment by PollyV1** [comment]
> Also, duplicate of #15774

i closed #15774 already.

File: keyboards/carpolly/readme.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# carpolly
+
+Keyboard Maintainer: Paul Enrico N. Viola (https://github.com/PollyV1)
+Hardware Supported: Teensy 2 controller
+Hardware Availability: https://github.com/PollyV1/Carpolly
+
+carpolly - a handwired keyboard on a teensy 2 controller
+
+https://imgur.com/w691sNn
```
Comment (drashna) [inline keyboards/carpolly/readme.md:9]:
```suggestion
# carpolly

![](https://i.imgur.com/w691sNnl.png)

A handwired keyboard on a teensy 2 controller

* Keyboard Maintainer: Paul Enrico N. Viola (https://github.com/PollyV1)
* Hardware Supported: Teensy 2 controller
* Hardware Availability: https://github.com/PollyV1/Carpolly

```

File: keyboards/carpolly/rules.mk [hunk 7]
```diff
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
```
Comment (drashna) [inline keyboards/carpolly/rules.mk:10]: There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

File: keyboards/carpolly/rules.mk [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = yes            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (drashna) [inline keyboards/carpolly/rules.mk:18]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/carpolly/readme.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+![](https://i.imgur.com/w691sNnl.png)
+
+A handwired keyboard on a teensy 2 controller
+
+* Keyboard Maintainer: Paul Enrico N. Viola (https://github.com/PollyV1)
+* Hardware Supported: Teensy 2 controller
+* Hardware Availability: https://github.com/PollyV1/Carpolly
+
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/carpolly/readme.md:12]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

File: keyboards/handwired/polly40/keymaps/polly40.json [hunk 10]
```diff
@@ -0,0 +1,109 @@
+{
```
Comment (drashna) [inline keyboards/handwired/polly40/keymaps/polly40.json:1]: This appears to be a VIA json. 
https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs

> Do not include VIA json files in the PR. These do not belong in the QMK repository as they are not used by QMK firmware – they belong in the [VIA Keyboard Repo](https://github.com/the-via/keyboards>

File: keyboards/handwired/polly40/keymaps/polly_40New.json [hunk 11]
```diff
@@ -0,0 +1,207 @@
+{
```
Comment (drashna) [inline keyboards/handwired/polly40/keymaps/polly_40New.json:1]: This appears to be a VIA json. 
https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs

> Do not include VIA json files in the PR. These do not belong in the QMK repository as they are not used by QMK firmware – they belong in the [VIA Keyboard Repo](https://github.com/the-via/keyboards>

File: keyboards/handwired/polly40/readme.md [hunk 12]
```diff
@@ -0,0 +1,15 @@
...
+
+42-keys ergonomic keyboard running on Adafruit Feather BLE.
+
+![42](https://github.com/nglgzz/42/raw/master/42.jpg)
+
+Keyboard Maintainer: [Angelo Gazzola (nglgzz)](https://github.com/nglgzz)  
+Hardware Supported: 42 (rev2, rev3), Feather 32u4 BLE  
+Hardware Availability: [GitHub](https://github.com/nglgzz/42/)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/handwired/polly40/readme.md:11]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

File: keyboards/handwired/polly40/rules.mk [hunk 13]
```diff
@@ -0,0 +1,25 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = halfkay
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
```
Comment (drashna) [inline keyboards/handwired/polly40/rules.mk:10]: There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

File: keyboards/handwired/polly40/rules.mk [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
```
Comment (drashna) [inline keyboards/handwired/polly40/rules.mk:22]:
```suggestion
NKRO_ENABLE = yes            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/polly40/Makefile [hunk 15]
```diff
@@ -0,0 +1,3 @@
+ifndef MAKEFILE_INCLUDED
```
Comment (drashna) [inline keyboards/polly40/Makefile:1]: please remove this file

File: keyboards/polly40/keymaps/polly40.json [hunk 16]
```diff
@@ -0,0 +1,109 @@
+{
```
Comment (drashna) [inline keyboards/polly40/keymaps/polly40.json:1]: This appears to be a VIA json. 
https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs

> Do not include VIA json files in the PR. These do not belong in the QMK repository as they are not used by QMK firmware – they belong in the [VIA Keyboard Repo](https://github.com/the-via/keyboards>

File: keyboards/polly40/keymaps/polly_40New.json [hunk 17]
```diff
@@ -0,0 +1,207 @@
+{
```
Comment (drashna) [inline keyboards/polly40/keymaps/polly_40New.json:1]: This appears to be a VIA json. 
https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs

> Do not include VIA json files in the PR. These do not belong in the QMK repository as they are not used by QMK firmware – they belong in the [VIA Keyboard Repo](https://github.com/the-via/keyboards>

File: keyboards/polly40/readme.md [hunk 18]
```diff
@@ -0,0 +1,15 @@
...
+
+42-keys ergonomic keyboard running on Adafruit Feather BLE.
+
+![42](https://github.com/nglgzz/42/raw/master/42.jpg)
+
+Keyboard Maintainer: [Angelo Gazzola (nglgzz)](https://github.com/nglgzz)  
+Hardware Supported: 42 (rev2, rev3), Feather 32u4 BLE  
+Hardware Availability: [GitHub](https://github.com/nglgzz/42/)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/polly40/readme.md:11]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

File: keyboards/polly40/rules.mk [hunk 19]
```diff
@@ -0,0 +1,25 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = halfkay
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
```
Comment (drashna) [inline keyboards/polly40/rules.mk:10]: There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

**Review by drashna — CHANGES_REQUESTED** [review]
it also looks like you have handwired/polly40, and polly40.  Is that intentional?

**Comment by PollyV1** [comment]
> it also looks like you have handwired/polly40, and polly40. Is that intentional?

Nope. That was my 1st firmware or my other keyboard. Im not pulling a request for that. Only for the carpolly. You may disregard polly40

**Comment by PollyV1** [comment]
I removed polly40 and handwired/polly40 already as well as the carpolly. Just working on handwired/carpolly. Thank you

**Comment by PollyV1** [comment]
> I removed polly40 and handwired/polly40 already as well as the carpolly. Just working on handwired/carpolly. Thank you

Im new to git, if i commit suggestions it edits and commits to my file already?

File: keyboards/handwired/carpolly/rules.mk [hunk 20]
```diff
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
```
Comment (drashna) [inline keyboards/handwired/carpolly/rules.mk:10]: There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/carpolly/readme.md [hunk 21]
```diff
@@ -0,0 +1,28 @@
+# carpolly
+
+![carpool](https://i.imgur.com/w691sNn.jpg[/img)
```
Comment (fauxpark) [inline keyboards/handwired/carpolly/readme.md:3]:
```suggestion
![carpool](https://i.imgur.com/w691sNn.jpg)
```

File: keyboards/handwired/carpolly/readme.md [hunk 22]
```diff
@@ -0,0 +1,28 @@
...
+
+Keyboard Maintainer: Paul Enrico N. Viola (https://github.com/PollyV1)<br />                                                                                                     Hardware Supported: Teensy 2 controller<br />                                                                                                                                     Hardware Availability: [PollyV1] https://github.com/PollyV1/Carpolly<br />
```
Comment (fauxpark) [inline keyboards/handwired/carpolly/readme.md:5]:
```suggestion
* Keyboard Maintainer: Paul Enrico N. Viola (https://github.com/PollyV1)
* Hardware Supported: Teensy 2 controller
* Hardware Availability: [PollyV1] https://github.com/PollyV1/Carpolly
```

File: keyboards/handwired/carpolly/readme.md [hunk 23]
```diff
@@ -0,0 +1,28 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    make carpolly/default:default
+
+## Flashing
+
+Flashing example for this keyboard:
+
+    make carpolly:default:flash
```
Comment (fauxpark) [inline keyboards/handwired/carpolly/readme.md:19]:
```suggestion
    make handwired/carpolly:default

## Flashing

Flashing example for this keyboard:

    make handwired/carpolly:default:flash
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/carpolly/readme.md [hunk 24]
```diff
@@ -0,0 +1,30 @@
+# carpolly
+
+![carpool](https://i.imgur.com/w691sNn.jpg)
```
Comment (fauxpark) [inline keyboards/handwired/carpolly/readme.md:3]:
```suggestion
![carpool](https://i.imgur.com/w691sNnh.jpg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
thanks!

**Comment by PollyV1** [comment]
Thank you guys!!!

**Pull request merged**